src/Components/CardsContainer.css

```
@@ -1,3 +1,6 @@
 .cardsContainer{
-    
+    margin:0 auto;
+}
+.dark .card{
+    background-color: #222f3e;
```

Maybe here we could use the team's dark grey official color
```suggestion
    background-color: #3c3d3b;
```